src/IDPDiscoveryController.js

```diff
@@ -66,6 +65,10 @@ function (Okta, PrimaryAuthController, PrimaryAuthModel, IDPDiscoveryForm, IDPDi
           this.options.appState.trigger('navigate', 'signin');
         }
       });
+
+      this.listenTo(this.model, 'goToOtherIdpAuth', function (url) {
+        Util.postToUrl(url, this.$el);
```

Comment:
just thinking... could we append the form to the `body` of the page so we avoid passing the extra `$el` param and maybe even the extra event `goToOtherIdpAuth`?

Reply:
good question and it's what I thought initially then I'm slightly leaning towards to not pollute the `body` especially when customer host the widget.
although it actually doesn't quite matter given we're leaving the page anyway after submit..

Reply:
yeah... at the end it doesn't matter... so I guess either way works, I was just thinking a way to avoid the event and then handling it here as opposed to do everything on the view... but I agree with the point of not polluting the page's body. So sounds good, we can keep it as you have it